email_alert_service/models/message_processor.rb

```diff
@@ -1,3 +1,5 @@
+require_relative('./message_processor_validator')
```

Comment:
Nitpick: Could this be moved to `major_change_message_processor.rb` as that's where the include is?

Reply:
Good point, needs to be repeated in both processor classes but that's much clearer :+1: